cerberus-artemis-kpi-lambda/README.md

```diff
@@ -0,0 +1,6 @@
+# Cerberus Artemis KPI Lambda
```

Comment:
I don't know that we should call it Artemis, especially if we aren't defining Artemis.  How about 'Cerberus Example KPI Lambda'?